<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true"/>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>170</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans [monotype]</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Allgemein</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>404</width>
               <height>118</height>
              </rect>
             </property>
             <widget class="QGroupBox" name="groupBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>0</y>
                <width>141</width>
                <height>101</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="title">
               <string>Read-Mode:</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <widget class="QRadioButton" name="radioButton">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>121</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>Offline lesen ...</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QRadioButton" name="radioButton_2">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>70</y>
                 <width>100</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>Online</string>
               </property>
              </widget>
             </widget>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="geometry">
               <rect>
                <x>170</x>
                <y>0</y>
                <width>221</width>
                <height>101</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="title">
               <string>Speicherort:</string>
              </property>
              <widget class="QLineEdit" name="lineEdit">
               <property name="geometry">
                <rect>
                 <x>12</x>
                 <y>32</y>
                 <width>151</width>
                 <height>25</height>
                </rect>
               </property>
               <property name="text">
                <string>/home/</string>
               </property>
              </widget>
              <widget class="QPushButton" name="pushButton_5">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>30</y>
                 <width>30</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
              <widget class="QCheckBox" name="checkBox">
               <property name="geometry">
                <rect>
                 <x>11</x>
                 <y>67</y>
                 <width>151</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>Daten packen</string>
               </property>
              </widget>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="font">
              <font>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Rückwärts Blättern</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Home</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="font">
              <font>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Vorwärts Blättern</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>News - Site</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <item>
          <property name="text">
           <string>AI</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>techcrunch.com</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mit.edu</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>wired.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>spectrum.ieee.org</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>nypost.com</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>machine learning</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>techcrunch.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mit.edu</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>wired.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>spectrum.ieee.org</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>computer vison</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>computer vison</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>... Auffrischen ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_2">
         <item>
          <property name="text">
           <string>Neues Element</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Neues Element</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Neues Element</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Neues Element</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Neues Element</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Neues Element</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="toolTip">
    <string>Programm Beenden</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>windowBecomesFocus()</signal>
   <receiver>MainWindow</receiver>
   <slot>showEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>windowBecomesFocus()</signal>
  <slot>showEvent()</slot>
 </slots>
</ui>
